lib/datasource/remote/api/proposals_repository.dart

```diff
@@ -66,24 +65,26 @@ class ProposalsRepository extends NetworkRepository with EosRepository {
         .catchError((error) => mapHttpError(error));
   }
 
-  Future<Result> getReferendums(ProposalType proposalType) {
-    print('[http] get referendums: stage = [${proposalType.filterByStage}]');
+  Future<Result> getReferendums(String scope, bool isReverse) {
+    print('[http] get referendums: stage = [$scope]');
 
     final request = createRequest(
       code: account_rules,
-      scope: proposalType.filterByStage ?? '',
+      scope: scope,
       table: tableReferendums,
       limit: 100,
-      reverse: proposalType.isReverse,
+      reverse: isReverse,
     );
 
     final proposalsURL = Uri.parse('$baseURL/v1/chain/get_table_rows');
 
     return http
         .post(proposalsURL, headers: headers, body: request)
         .then((http.Response response) => mapHttpResponse(response, (dynamic body) {
+              // The referendums do not have a status field as do the proposals, so the scope must be added
+              // to each referendum, which also acts as a status field.
               final List<ReferendumModel> result =
-                  body['rows'].map<ReferendumModel>((i) => ReferendumModel.fromJson(i)).toList();
+                  body['rows'].map<ReferendumModel>((i) => ReferendumModel.fromJson(i, scope)).toList();
```

Comment:
@n13 It would be easier if the referendum had a `status` field, so I would avoid having to pass an extra field to each referendum to show if it is `passed` or `failed` in the UI.

Reply:
I know but for the time being we don't have it, we only have what we have. 

Not changing it since all this is changing when we move to dao.seeds for both proposals and referendums.

Then all the data structures change anyway

Reply:
The referendums architecture is not great BTW, I am aware, which is another reason we were refactoring it all into dao.seeds, that removes the inconsistencies. For the time being, we work with the old data structure.